opentracing-api-extensions-tracer-spring-autoconfigure/pom.xml

```diff
@@ -19,7 +19,7 @@
   <parent>
     <artifactId>opentracing-api-extensions-parent</artifactId>
     <groupId>io.opentracing.contrib</groupId>
-    <version>0.4.1-SNAPSHOT</version>
+    <version>0.5.0-SNAPSHOT</version>
```

Comment:
Could you revert the version changes (here and other poms) back to 0.4.1 - the release process will automatically modify the versions.